ci-operator/step-registry/openshift-qe/installer/bm/day2/regulus/openshift-qe-installer-bm-day2-regulus-commands.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+# Install, config and execute Regulus, the dataplane test suite.
+#
+# Background:
+#   For BM we support multiple profiles. Currently there are 2 profiles: metal-perscale-cpt (cloud31)
+#   and metal-perfscale-jetlag (cloud48). More profiles are coming. Multiple profiles ramifications are:
+#       1. The current BM Prow design has cloud31 as the only cloud reachable from Prow pod.
+#       2. Dictated by "1.", jetlag always runs on cloud31 bastion to instantiatep cloud31 or cloud48 cluster.
+#   Notable impacts to the Regulus Step are:
+#      If the profile is cloud31, we ssh from Prow to cloud31's bastion and run Regulus there, a standard case.
+#      If the profile is cloud48, we have to double/nested ssh to get to cloud48's bastion. A more complicate situation.
+#      For this 2-hop scenario, we refer to cloud31's bastion as "bastion" and the second hop bastion as "true" bastion.
+#      On the true bastion, we do not have the private key to ssh to its nodes (ssh core@worker_ip). This key
+#      is on the bastion (where jetlag ran).
+#
+#      In summary, for Regulus some operations are 1-hop ssh, some are 2-hop ssh, and some need jetlag build-time
+#      private key that we need to set up.
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+REPO_NAME=${REPO_NAME:-}
+PULL_NUMBER=${PULL_NUMBER:-}
+BASTION="${BASTION:-}"                          # first level which always is cloud31 bastion
+TRUE_BASTION_HOST="${TRUE_BASTION_HOST:-}"      # second level such as cloud48 bastion
+TRUE_BASTION_USER="${TRUE_BASTION_USER:-root}"
```

Comment:
Do we need this? It's always root on all the perf/scale lab allocations

Reply:
This is for Bring-Your-Own-Lab. I use Regulus with JetSki-built cluster too. If this will never happen with ci-tools, I can move this var to inside Regulus layer.